src/ssd/resize_indicator.c

```diff
@@ -111,27 +144,40 @@ void
 resize_indicator_update(struct view *view)
 {
 	assert(view);
+	assert(view == view->server->grabbed_view);
+
+	if (!wants_indicator(view)) {
+		return;
+	}
+
+	/*
+	 * Following asserts ensure that we'll never get out-of-sync
```

Comment:
If there is a chance that the arguments passed to the asserts may be false because of something the user does (like re-loading the config) then we should re-think approach.
Sorry if I misunderstood logic :smile:

Reply:
Comment is badly formulated, will change.

Those asserts() should obviously never trigger by anything the users does, they will however trigger if we do something bad in the future :)

Reply:
Thought it was just the comment.
:smile:

Reply:
I have reconsidered the `assert()` approach for this case: it now works around it and logs an `WLR_INFO` message that states that resize_indicator has to use a fallback path. I imagine changes in the future might not get necessarily tested with all variations of the `rc.resize_indicator` setting and thus the chances of a bug slipping through is non-zero in this case. This change prevents crashing labwc for users in that case and should still allow devs to detect the issue after the fact via the log message.